app/controllers/application_controller.rb

```diff
@@ -21,4 +21,14 @@ def error500(e)
     logger.error [e, *e.backtrace].join("\n")
     render 'error500', status: 500, formats: [:html]
   end
+
+  before_filter :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.for(:sign_up) { |u| u.permit(:username, :email, :password, :password_confirmation, :remember_me) }
+    devise_parameter_sanitizer.for(:sign_in) { |u| u.permit(:login, :username, :email, :password, :remember_me) }
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [113/100] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)